apps/frontend/src/components/global/upload_tabs/splunk/SplunkReader.vue

```diff
@@ -49,6 +49,31 @@
             <v-icon pr-2>mdi-github-circle</v-icon>
             Splunk Configuration
           </v-btn>
+          <br />
+          <span>
+            The Splunk platform accepts any type of data. In particular, it
+            works with all IT streaming
+            <br />
+            and historical data. The source of the data can be event logs, web
+            logs, live application logs,
+            <br />
+            network feeds, system metrics, change monitoring, message queues,
+            archive files, and so on.
+          </span>
+          <br />
+          <span>
+            <br />
+            "A Splunk index is a repository for Splunk data."
+            <br />
+            Data that has not been previously added to Splunk is referred to as
+            raw data.
+            <br />
+            When the data is added to Splunk, it indexes the data (uses the data
```

Comment:
the parenthetical clause doesn't add any new information/clarification.

Reply:
yes it does, it informs the user how indexes are created.